src/Client/Campaign.php

```diff
@@ -16,6 +16,15 @@
  *      id: string,
  *      name: string,
  *      logoUri: ?string,
+ *      phoneNumber: ?string,
+ *      emailAddress: ?string,
```

Comment:
this is technically a lie - sf doesn't actually return emailAddress here yet, but I think we need to add it. I'll do that next and make matchbot tolerate it being missing.